cli/pkg/cmd/search/testdata/TestSearch.golden

```diff
@@ -0,0 +1,3 @@
+NAME            KIND       DESCRIPTION                                  TAGS
+foo (0.1)       Task       Description for task abc version 0.1         tag3, tag1   
+foo-bar (0.2)   Pipeline   Description for pipeline foo-bar versio...   ---          
```

Comment:
Should "versions" be on there own column ?

Reply:
yeah initially we added a separate column for the version `LATEST VERSION`, it was taking too much space :smile: so we appended that to name. should we have a separate column? WDYT?

Reply:
yes, the same reasoning as @SM43 .. does `name (ver)` hinder readabilty?